### ISSUE TITLE
can't clear intent in console with OrientDB 2.1.9

### ISSUE THREAD

**Issue by lebedov** [orientechnologies/orientdb#5601]
According to the [online documentation](http://orientdb.com/docs/2.1/Console-Command-Declare-Intent.html), one should be able to clear a set intent at the console using DECLARE INTENT NULL. When I tried doing so with OrientDB 2.1.9, however, the following error occurred:

```
Declaring intent 'NULL'...
Error: java.lang.IllegalArgumentException: Intent 'null' not supported. Available ones are: massiveinsert, massiveread
```

**Comment by luigidellaquila** [comment]
Hi @lebedov 

I have a fix for this in a local branch, it will be merged in 2.1.x asap.
Unfortunately I won't be able to include it in 2.1.10 because the release validation process is already running, so it will be in 2.1.11

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @lebedov 

good news, this fix was merged in 2.1.x, so it will be released with 2.1.10

Thanks

Luigi

**Comment by lebedov** [comment]
Grazie, @luigidellaquila !
